### DISCUSSION TITLE
v7.4.0附录中的参考文献无法正常显示

### DISCUSSION THREAD

**Discussion by chelungungun0** [tuna/thuthesis#890] [Q&A] [answered]
从[TUNA 镜像站]下载的v7.4.0版本模板，发现其中自带的thuthesis-example.pdf中，附录的参考文献索引显示的是[?]，且没有生成附录参考文献。使用TexStudio对thuthesis-example.tex进行编译，所有设置均为默认，得到的pdf具有相同的问题。

**Comment by zepinglee** [comment]
大概是因为 l3build 的编译流程默认没有用 bibtex 编译附录的参考文献。应该可以在 l3build 的配置中解决。

**Reply by zepinglee** [reply, marked as answer]
在 [ea7722a](https://github.com/tuna/thuthesis/commit/ea7722ac2bb504ac79b74a829314479d2ceb32f8) 中解决。

**Reply by chelungungun0** [reply]
谢谢！

**Comment by chelungungun0** [comment]
在v7.4.0版本里，后来我试了一下通过命令提示符使用latexmk编译，发现附录的参考文献正常了，而且附录写好以后只要用latexmk编译一次，texstudio就可以正常编译了。希望可以帮到遇到同样问题的同学。

**Reply by dgqblyk** [reply]
我在使用latexmk编译之后，texstudio编译依然存在问题。而且上面的l3build配置我也没有看懂怎么搞...@zepinglee

**Reply by zepinglee** [reply]
新开 issue 详细描述你的问题
